tools/common.ps1

```diff
@@ -85,7 +85,7 @@ function Get-VsTestPath {
     # Unfortunately CI doesn't add vstest to PATH. Test existence of vstest
     # install paths instead.
 
-    $ManualVsTestPath = "$(Split-Path $PSScriptRoot -Parent)\artifacts\Microsoft.TestPlatform\tools\net451\Common7\IDE\Extensions\TestPlatform"
+    $ManualVsTestPath = "$(Split-Path $PSScriptRoot -Parent)\build\Microsoft.TestPlatform\tools\net451\Common7\IDE\Extensions\TestPlatform"
```

Comment:
I wasn't sure about this one.

Reply:
IMO build should be only artifacts built in this repo. The artifacts dir for external dependencies is good IMO.

Reply:
Similar to the other comment thread, this goes to standardizing build, outputs and packaging (for ingestion); and applies to all projects leveraging `microsoft/undocked`. This _might_ be a bit easier to add a custom knob to change `build` to `artifacts` but then the knob must be added to the `build.yml` template as well. I'll take a look at adding that support.

Reply:
BTW, this directory contains both a `bin` and `obj` subdirectory. So it is all build output.

Reply:
https://github.com/microsoft/undocked/pull/11